pkg/utils/utils.go

```diff
@@ -65,3 +68,21 @@ func CopyFile(src, dst string) (int64, error) {
 	n, err := io.Copy(destination, source)
 	return n, err
 }
+
+// getTLSConfig get tls config from CA, Cert and Key file
+func GetTLSConfig(caCertPath, certPath, keyPath string) (*x509.CertPool, tls.Certificate, error) {
+	cert, err := tls.LoadX509KeyPair(certPath, keyPath)
+	if err != nil {
+		return nil, tls.Certificate{}, err
+	}
+
+	caCert, err := ioutil.ReadFile(caCertPath)
```

Comment:
ioutil is deprecated.
https://go.dev/doc/go1.16#ioutil

Reply:
okay, i will change it to `os.ReadFile`